console/src/components/Services/Data/DataSubSidebar.js

```diff
@@ -108,10 +111,23 @@ class DataSubSidebar extends React.Component {
             ) {
               activeTableClass = styles.activeTable;
             }
+            const schemeWarning = !gqlPattern.test(tableName) ? (
```

Comment:
rename variable to `gqlCompatibilityWarning`. Move the logic to a common function called `getGqlCompatibilityWarning` outside the `.map`. Will make reading code much easier